clients/hadoopfs/src/main/java/io/lakefs/LakeFSFileSystem.java

```diff
@@ -188,6 +188,7 @@ public long getDefaultBlockSize() {
 
     @Override
     public FSDataInputStream open(Path path, int bufSize) throws IOException {
+        LOG.trace("open_op: {}", path);
```

Comment:
Why the `_op` suffix?

Consider just logging the call, e.g.
```suggestion
        LOG.trace("open({})", path);
```

Reply:
The reason why I added the op is to make these log messages searchable, if you don’t think that’s required I’m happy to remove it!

Reply:
As discussed f2f, I added another operation logger with a searchable prefix.